automation/publish.sh

```diff
@@ -6,9 +6,34 @@
 # IMAGE_REPO
 # To run it just do proper docker login and automation/publish.sh
 
+image_registry=${IMAGE_REGISTRY:-quay.io}
+image_repo=${IMAGE_REPO:-nmstate}
+
+# Publish kubernets-nmstate containers from master
 source automation/check-patch.setup.sh
-cd ${TMP_PROJECT_PATH}
-make \
-    IMAGE_REGISTRY=${IMAGE_REGISTRY:-quay.io}  \
-    IMAGE_REPO=${IMAGE_REPO:-nmstate} \
-    push-handler
+(
+    cd ${TMP_PROJECT_PATH}
+    make \
+        IMAGE_REGISTRY=${image_registry}  \
+        IMAGE_REPO=${image_repo} \
+        push-handler
+)
+
+# Update kubernetes-nmstate documentation container
+docs_container=${image_registry}/${image_repo}/kubernetes-nmstate-docs:master
+docker build docs --build-arg BRANCH=master -t $docs_container
+docker push $docs_container
+
+# To pass user/password from automations, idea is taken from [1]
+# [1] https://stackoverflow.com/questions/8536732/can-i-hold-git-credentials-in-environment-variables
+git config credential.helper '!f() { sleep 1; echo "username=${GITHUB_USER}"; echo "password=${GITHUB_TOKEN}"; }; f'
```

Comment:
Would this hide our secrets? Would you set `set +x` to make extra sure to hide it?

Reply:
We have the same thing here, I think the password does not appear https://github.com/nmstate/kubernetes-nmstate/blob/master/automation/release.sh#L10.